Jenkinsfile

```diff
@@ -4,5 +4,4 @@ REPOSITORY = 'government-frontend'
 
 node {
   def govuk = load '/var/lib/jenkins/groovy_scripts/govuk_jenkinslib.groovy'
-  govuk.buildProject(publishingE2ETests: true)
```

Comment:
removing this will actually remove the whole build, what you'd want to do is: `govuk.buildProject()`